incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -344,6 +345,57 @@ It means that this namespace is orphaned and its parent has been deleted. To fix
 $ kubectl hns set --root <namespace>
 ```
 
+<a name="use-limit-propagation"/>
+
+### Limit the propagation of an object to descendant namespaces
+
+To limit the propagation of an object, you can use any of the following exceptions selectors:
+
+```
+propagate.hnc.x-k8s.io/select 
+```
+The value for this selector has to be a valid k8s label selector. The object will only be propagated to namespaces whose labels match the label selector.
```

Comment:
This looks a bit weird. Maybe put these all in a list? E.g.:

* **`propagate.hnc.x-k8s.io/select`**: The object will only be propagated to namespaces whose labels match the label selector. The value...
* **`propagate.hnc.x-k8s.io/treeSelect`**: Use namespace _names_ to represent where this object should be propagated, ...

In Markdown, this looks like this:
```
* **`propagate.hnc.x-k8s.io/select`**: The object will only be propagated to namespaces whose labels match the label selector. The value...
* **`propagate.hnc.x-k8s.io/treeSelect`**: Use namespace _names_ to represent where this object should be propagated, ...
```

Reply:
Also for that first one (`/select`): I'd focus first on what the annotation _does_ (propagate to namespaces with labels), and secondly on _how_ it does it (it must be a valid k8s label selector).

Finally, can you spell out k8s (e.g. "valid Kubernetes label selector") and link to the correct part of the K8s docs? https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors